cmd/kube-apiserver/app/options/options.go

```diff
@@ -49,7 +50,19 @@ func NewAPIServer() *APIServer {
 		ServerRunOptions: genericoptions.NewServerRunOptions().WithEtcdOptions(),
 		EventTTL:         1 * time.Hour,
 		KubeletConfig: kubeletclient.KubeletClientConfig{
-			Port:        ports.KubeletPort,
+			Port: ports.KubeletPort,
+			PreferredAddressType: []string{
```

Comment:
Need to rebase, this bit already merged, but the reason is that these config options are bound to a stringslice CLI flag